packages/kbn-esql-validation-autocomplete/src/definitions/builtin.ts

```diff
@@ -24,28 +24,42 @@ function createMathDefinition(
     description,
     supportedCommands: ['eval', 'where', 'row', 'stats', 'metrics', 'sort'],
     supportedOptions: ['by'],
-    signatures: types.map((type) => {
-      if (Array.isArray(type)) {
-        return {
-          params: [
-            { name: 'left', type: type[0] },
-            { name: 'right', type: type[1] },
-          ],
-          returnType: type[2],
-        };
-      }
+    signatures: functionSignatures.map((functionSignature) => {
+      const [lhs, rhs, result] = functionSignature;
       return {
         params: [
-          { name: 'left', type },
-          { name: 'right', type },
+          { name: 'left', type: lhs },
+          { name: 'right', type: rhs },
         ],
-        returnType: type,
+        returnType: result,
       };
     }),
     validate,
   };
 }
 
+// https://www.elastic.co/guide/en/elasticsearch/reference/master/esql-functions-operators.html#_less_than
+const baseComparisonTypeTable: MathFunctionSignature[] = [
```

Comment:
Great work sorting this out. It will be interesting to see how this compares to the generated definitions when we do https://github.com/elastic/kibana/issues/188978